string      gesture_type
uint8       gesture_id
int64       gesture_duration
uint32      bow_angle
uint32      nod_angle
float64     location_x
float64     location_y
float64     location_z